### PR TITLE
minor adjustments to multiple pages

### PR DESCRIPTION
**Pull request by wendidy**
Please refer to the points that are crossed off in this document: https://docs.google.com/document/d/1GLshBv7Ue09ceCFDSLmFvrgbEofGkY6R0jXlKxVjqHc/edit?ts=607ded9d

Screenshots(only some minor changes):
1. Added a side scrollbar for cards that have too much content(only visible when scrolling):
image

other things are pretty straight forward, if screenshots are needed then I will insert them here

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1341,3 +1341,12 @@ hr {
   margin: 0.5rem 0;
   border: 1px solid #dfe0e2;
 }
+
+.orgLogo {
+  height: 50px;
+  object-fit: scale-down;
+}
+
+.hidden-overflow {
```
Comment (garyluu) [inline src/styles.scss:1350]: try:

```
@each $value in $overflows {
  .overflow-#{$value} { overflow: $value !important; }
}
```
and using class overflow-hidden and see what happens. if it works, then ur .content style can also be replaced

assuming everything works, put that in  the boostrap4 scss file

Comment (wendidy) [reply src/styles.scss:1350]: It does, thank you!

File: src/app/about/about.component.html [hunk 2]
```diff
@@ -36,29 +36,29 @@ <h5>
...
 
     <div fxLayout="column" class="container" fxLayoutAlign="space-between center" fxLayoutGap="1rem">
-      <h3>Dockstore follows the FAIR Guiding Principles</h3>
+      <h2>Dockstore follows the FAIR Guiding Principles</h2>
       <span
         >Our
         <a
           target="_blank"
           rel="noopener noreferrer"
-          [href]="Dockstore.DOCUMENTATION_URL + 'advanced-topics/best-practices/best-practices-secure-fair-workflows.html'"
```
Comment (garyluu) [inline src/app/about/about.component.html:45]: Assuming this is intended

**Review by garyluu — APPROVED** [review]
Overall assuming everything looks right

File: src/app/about/about.component.html (hunk 2 above)
Comment (denis-yuen) [reply src/app/about/about.component.html:45]: Hard-coding this does seem a bit odd

**Review by denis-yuen — APPROVED** [review]
Sonarcloud smells are intentional, removal of DOCUMENTATION_URL seems odd

File: src/app/about/about.component.html (hunk 2 above)
Comment (denis-yuen) [reply src/app/about/about.component.html:45]: In discussion, oddly https://docs.dockstore.org/en/develop/advanced-topics/best-practices/best-practices-secure-fair-workflows.html exists but https://docs.dockstore.org/en/latest/advanced-topics/best-practices/best-practices-secure-fair-workflows.html does not

Comment (denis-yuen) [reply src/app/about/about.component.html:45]: Probably due to https://ucsc-cgl.atlassian.net/browse/SEAB-2737

**Pull request merged**